src/ResourceManager/Sql/Commands.Sql/Auditing/Model/AuditingPolicyModel.cs

```diff
@@ -46,11 +46,6 @@ public abstract class AuditingPolicyModel
         public string ServerName { get; set; }
 
         /// <summary>
-        /// Gets or sets the audit type
-        /// </summary>
-        public AuditType AuditType { get; set; }
```

Comment:
Never mind, I see you added it to DatabaseAuditingPolicyModel.cs and ServerAuditingPolicyModel.cs